packages/manager/apps/public-cloud/src/links.json

```diff
@@ -337,7 +347,7 @@
     },
     "redirect": {
       "application": "public-cloud",
-      "path": "/pci/projects/{project}/storages/databases-analytics/data-analysis/new?e=grafana"
+      "path": "/pci/projects/{project}/databases-analytics/analysis/services/new?STEP_1=dashboard"
```

Comment:
Due to technical limitations, URLs do not reflect the changes in engines names, so we have to match the one used by the backend:
`/pci/projects/{project}/databases-analytics/analysis/services/new?STEP_1=grafana`